dev/date-picker.html

```diff
@@ -9,6 +9,9 @@
     <script type="module">
       import '@vaadin/date-picker';
       import '@vaadin/tooltip';
+      const isDateDisabled = (date) => !!(date?.getDay() === 0);
```

Comment:
This is a sample implementation in the demo app. We can drop this or add some docs as needed to describe what's going on.